File: C Programming/Assignments/Assignment 0/LeapYear.c
#include<stdio.h>
#include<conio.h>
int main()
{
int year;
printf("Enter Year=");
scanf("%d",&year);
if(year%4==0){
	printf("Leap Year");
}
else{
	printf("Non-Leap Year");
}
return 0;
}

File: JAVA/Assignment 0/Leapyear.java
import java.util.Scanner;
public class Leapyear
{
public static void main(String args[])
{
Scanner sc=new Scanner(System.in);
int year;
System.out.printf("Enter Year=");
year=sc.nextInt();

if(year%4==0 && year%100==0 || year%400==0){
	System.out.printf("Leap Year");
}
else{
	System.out.printf("Non-Leap Year");
}
}
}